Data/ByteString/Builder.hs

```diff
@@ -478,3 +478,6 @@ stringUtf8 = P.primMapListBounded P.charUtf8
 
 instance IsString Builder where
     fromString = stringUtf8
+
+instance Show Builder where
+    showsPrec _ = shows . toLazyByteString
```

Comment:
If you can afford to ignore the precedence (which is the case), there is no benefit in `showsPrec`: it is enough to define `show` instead.

Reply:
Done in fd464d3